src/include/job.h

```diff
@@ -527,6 +527,7 @@ enum bg_hook_request {
 	BG_PBSE_SISCOMM,
 	BG_IM_DELETE_JOB_REPLY,
 	BG_IM_DELETE_JOB,
+	BG_IM_DELETE_JOB2,
```

Comment:
Is this entry as descriptive as it could be? What distinguishes BG_IM_DELETE_JOB from BG_IM_DELETE_JOB2?

Reply:
This is the newly created background request that a corresponds to IM_DELETE_JOB2 which is a well defined interface that was introduced with the release nodes feature.

Reply:
I'm not a fan of adding a numeric suffix to distinguish things that are similar, but different. I much prefer a more descriptive name. Since you are simply mirroring the IM_DELETE_JOB2 name, there is really no other option here.